package-manifest.toml

```diff
@@ -55,6 +55,15 @@ to = "/opt/oxide/cockroachdb/sql"
 from = "smf/cockroachdb"
 to = "/var/svc/manifest/site/cockroachdb"
 
+[package.internal-dns]
+rust.binary_names = ["dns-server"]
```

Comment:
Should `dnsadm` be here too, or do the associated CLI tools typically not go here?